lexlogic-web/next.config.mjs

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  images: {
+    domains: ['user-images.githubusercontent.com'],
```

Comment:
What is this domain, and what does it do?

Reply:
I'll remove that. It was so I could use an external image source but I'm not using that anymore